addon/find-role.js

```diff
@@ -1,5 +1,9 @@
 export default function findRole(...roles) {
   const selector = roles.map((role) => `[data-role="${role}"]`).join(' ');
 
+  if()   {
+    function () { }
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.<br>Missing name in function declaration.<br>'function' is defined but never used.